tests/keras_autolog/test_keras_autolog.py

```diff
@@ -12,14 +12,22 @@
 from unittest.mock import patch  # noqa
 
 
+@pytest.fixture(autouse=True)
+def clear_session():
+    yield
+    # Release the global state managed by Keras to avoid this issue:
+    # https://github.com/keras-team/keras/issues/2102
+    keras.backend.clear_session()
```

Comment:
This issue occurs in old versions (< 2.3) of keras:

In the cross version tests,

- test (keras / 2.3.1 / autologging): took **3m 27s**
- test (keras / 2.2.5 / autologging): took **16m 9s**

https://github.com/mlflow/mlflow/runs/1562184382?check_suite_focus=true